DataFormats/Headers/include/Headers/HeartbeatFrame.h

```diff
@@ -74,24 +83,26 @@ struct HeartbeatTrailer
     // the complete 64 bit trailer word, initialize with blockType 5 and size 1
     uint64_t trailerWord = 0x5100000000000000;
     struct {
-	// bit 0 to 31: data length in words
-	uint32_t dataLength;
-	// bit 32 to 52: detector specific status words
-	uint32_t status:21;
-	// bit 53: =1 in case a new physics trigger arrived within read-out period
-	uint16_t hbfTruncated:1;
-	// bit 54: =0 HBF correctly transmitted
-	uint16_t hbfStatus:1;
-	// bit 55: =1 HBa/0 HBr received
-	uint16_t hbAccept:1;
-	// bit 56 to 59: trailer length
-	uint16_t trailerLength:4;
-	// bit 60 to 63: block type (=5 HBF Trailer)
-	uint8_t blockType:4;
+        // bit 0 to 31: data length in words
+        uint32_t dataLength;
+        // bit 32 to 52: detector specific status words
+        uint32_t status:21;
```

Comment:
why these changes?

Reply:
@matthiasrichter ...

Reply:
tabs has been replaced with blanks

Reply:
the old and new indentations are not consistent with clang-format. But, given that we will fix this globally at some point I will merge this now.